scripts/MAZE-EN/c100206004.lua

```diff
@@ -0,0 +1,64 @@
+--Gate Guardian of Thunder and Wind
+--scripted by JoyJ
+function c100206004.initial_effect(c)
+	--fusion material
+	c:EnableReviveLimit()
+	aux.AddFusionProcCode2(c,25955164,62340868,true,true)
+	aux.AddContactFusionProcedure(c,Card.IsAbleToRemoveAsCost,LOCATION_ONFIELD+LOCATION_HAND+LOCATION_REMOVED,0,Duel.Remove,POS_FACEUP,REASON_COST)
+	--to hand
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(100206004,1))
+	e1:SetCategory(CATEGORY_TOHAND+CATEGORY_SEARCH)
+	e1:SetType(EFFECT_TYPE_IGNITION)
+	e1:SetRange(LOCATION_MZONE)
+	e1:SetCountLimit(1,100206004)
+	e1:SetTarget(c100206004.thtg)
+	e1:SetOperation(c100206004.thop)
+	c:RegisterEffect(e1)
+	--special summon
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(100206004,2))
+	e2:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e2:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+	e2:SetCode(EVENT_LEAVE_FIELD)
+	e2:SetProperty(EFFECT_FLAG_DELAY+EFFECT_FLAG_DAMAGE_STEP)
+	e2:SetCondition(c100206004.spcon)
+	e2:SetTarget(c100206004.sptg)
+	e2:SetOperation(c100206004.spop)
+	c:RegisterEffect(e2)
+end
+function c100206004.thfilter(c)
+	return (aux.IsCodeListed(25955164) or aux.IsCodeListed(62340868)
+		or aux.IsCodeListed(98434877)) and c:IsAbleToHand()
```

Comment:
```suggestion
	return (aux.IsCodeListed(c,25955164) or aux.IsCodeListed(c,62340868)
		or aux.IsCodeListed(c,98434877)) and c:IsAbleToHand()
```

Reply:
This one is weird... I prefer to use "or" because yugioh-wiki shows that the tempotary Japanese translation [いずれかの] (one of those).
Still, thanks for fixing the problem in aux.IsCodeListed.